Source/WebCore/platform/mediastream/WebRTCProvider.cpp

```diff
@@ -231,6 +361,24 @@ void WebRTCProvider::createDecodingConfiguration(MediaDecodingConfiguration&& co
         }
     }
 #endif
+    // For power efficient decoders, we use the regular media engine MC code path which has more fine grained checks.
+    if (info.powerEfficient && info.supportedConfiguration.video) {
+        auto videoConfiguration = info.supportedConfiguration;
+        videoConfiguration.audio = { };
+        videoConfiguration.video->contentType = contentTypeFromRTPVideoMimeType(info.supportedConfiguration.video->contentType);
```

Comment:
let's rename `supportedConfiguration` to `configuration` or in. follow-up change

Reply:
It will be done in a separate PR where we change the IDL as well.